NumLib/ODESolver/TimeDiscretization.h

```diff
@@ -142,6 +142,17 @@ class TimeDiscretization
     virtual void pushState(const double t, GlobalVector const& x,
                            InternalMatrixStorage const& strg) = 0;
 
+    /*! Indicate that the current timestep is rejected and that the computation
+     *  of the present time step will be restarted with a new time step size.
+     *
+     * \param t    The current time step.
```

Comment:
removed `t` by amending the commit of ``[dt] Set current x to old x when a step is ...``.